CMakeLists.txt

```diff
@@ -634,7 +634,8 @@ elseif(ANDROID)
 
   # Dynamically linked targets output libraries in raw/lib/<ABI> for easy including in apk with aapt
   set_target_properties(
-    lovr ${LOVR_ODE}
+    lovr
+    ${LOVR_ODE}
```

Comment:
Splitting this across two lines is not strictly relevant to oculus audio but would help a lot with forks (because it makes it possible to add additional items here without causing a merge conflict)